pkg/devices/cpu_power_meter.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package devices
+
+// EnergyZone represents a measurable energy or power zone/domain exposed by a power meter.
+// An EnergyZone typically represents a logical zone of the hardware unit, e.g. cpu core, cpu package
+// dram, uncore etc.
+// Reference: https://firefox-source-docs.mozilla.org/performance/power_profiling_overview.html
+type EnergyZone interface {
```

Comment:
You mention power in the Energy Zone. Currently, I only see values for Energy (joules). Should we also add  Power (Watts) variable in this PR as well?

Reply:
@KaiyiLiu1234, good question, I think we can leave that decision for later when `watts` is computed. 

I also think `Joules` and `Watts` should be in a separate `types` or `units` package but these changes can happen later.